docs/src/misc/notation.md

```diff
@@ -9,6 +9,7 @@ Within the documented functions, the utf8 symbols are used whenever possible, as
 
 | Symbol | Description | Also used | Comment |
 |:--:|:--------------- |:--:|:-- |
+| $ℝ, ℂ, ℍ$ | the set of real, complex and quaternion numbers, respectively. |
```

Comment:
Shouldn't this respect the alphabetical ordering of the table?

Reply:
I was not really sure, where to put those (and did not feel like doing three lines). Where do you propose to put this line?

Reply:
Real and complex numbers are already under "field", so maybe comment in that row could just be expanded a bit?

Reply:
ah, thanks for noticing, did exactly that.